test/framework/namespace.go

```diff
@@ -32,7 +33,7 @@ func generateNamespace(name string) *corev1.Namespace {
 
 // IBMCloudSecretName secret provided to enable access to test images
 // https://github.com/IBM/portieris/issues/34 to remove the need for this
-const IBMCloudSecretName = "default-icr-io"
+var IBMCloudSecretNames = []string{"default-icr-io", "all-icr-io"}
```

Comment:
I'm not quite clear on this - will follow up with Alex offline.

Reply:
The tests look for your pull secrets and copy them into new test namespace for you. On newer clusters, default-icr-io doesn't exist so I updated the test to try both.